src/status_im/contexts/profile/login/events.cljs

```diff
@@ -177,12 +177,13 @@
  :profile.login/get-user-password-success
  (fn [{:keys [db]} [password]]
    (when password
-     (let [{:keys [key-uid password]} (:profile/login db)]
-       {:db (-> db
-                (assoc-in [:profile/login :password] password)
-                (assoc-in [:profile/login :processing] true))
-        :fx [[:set-root :progress]
-             [:effects.profile/login [key-uid (security/safe-unmask-data password)]]]}))))
+     {:db (-> db
```

Comment:
but now there is no change? how is it fixed then?

Reply:
event `get-user-password-success` was called with right password. But in let block we were overwriting it with nil value

![Uploading image.png…]()